src/Illuminate/Console/Command.php

```diff
@@ -195,33 +195,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * Call another console command.
      *
-     * @param  string  $command
+     * @param  string|\Symfony\Component\Console\Command\Command  $command
      * @param  array   $arguments
      * @return int
      */
     public function call($command, array $arguments = [])
     {
-        $arguments['command'] = $command;
```

Comment:
Why do we need to manually set this argument? I've removed it and tried different commands. Everything seems to work.

If it wouldn't be removed - we cannot call FQCN names, because this argument not exists in command.